op-e2e/faultproofs/output_cannon_test.go

```diff
@@ -73,6 +74,79 @@ func TestOutputCannonGame(t *testing.T) {
 	game.WaitForGameStatus(ctx, disputegame.StatusChallengerWins)
 }
 
+func TestOutputCannon_ReclaimBond(t *testing.T) {
+	// The dishonest actor always posts claims with all zeros.
+	op_e2e.InitParallel(t, op_e2e.UsesCannon)
+	ctx := context.Background()
+	sys, l1Client := startFaultDisputeSystem(t)
+	t.Cleanup(sys.Close)
+
+	disputeGameFactory := disputegame.NewFactoryHelper(t, ctx, sys)
+	game := disputeGameFactory.StartOutputCannonGame(ctx, "sequencer", 3, common.Hash{})
+	game.LogGameData(ctx)
+
+	// The dispute game should have a zero balance
+	balance := game.WethBalance(ctx, game.Addr())
+	require.Zero(t, balance.Uint64())
+
+	alice := sys.Cfg.Secrets.Addresses().Alice
+	bal, _ := big.NewInt(0).SetString("1000000000000000000000", 10)
+	require.Equal(t, bal, game.Balance(ctx, alice))
+
+	// Grab the root claim
+	claim := game.RootClaim(ctx)
+	game.StartChallenger(
+		ctx,
+		"sequencer",
+		"Challenger",
+		challenger.WithPrivKey(sys.Cfg.Secrets.Alice),
+		challenger.WithClaimants(alice),
```

Comment:
Ah yep sorry about that, remnants of me trying to debug what was happening... I thought it might have just been overridden somewhere and wanted to check if being explicit would solve it :)